src/components/Cards/CharacterCard/CharacterCard.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import {View, Text, ImageBackground} from 'react-native';
+import {LinearGradient} from 'react-native-gradients';
+
+import styles from './CharacterCard.style';
+
+export default function CharacterCard({title, image}) {
+  const colorList = [
+    {offset: '35%', color: '#000', opacity: '0.65'},
+    {offset: '100%', color: '#000', opacity: '0'},
+  ];
```

Comment:
Bunlar sabit ise component'in dışına tanımlayın. Yoksa her render işleminde yeniden yeniden tanımlanır